config/initializers/assets.rb

```diff
@@ -12,3 +12,5 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+
+Rails.application.config.assets.paths << Rails.root.join('app', 'assets', 'fonts')
```

Comment:
Wondering if this line could be combined with the one on line 9 for node modules?